specklesystems/speckle-server
# feat(frontend-2): deploy fe2 as default on docker-compose environments

## Description & motivation

Frontend 2 is now available and can be deployed to Kubernetes (Helm) environments. Speckle also supports Docker Compose and we should enable Frontend 2 on those environments.

## Changes:

- In DigitalOcean 1-click environments, will deploy frontend-2 as default. Updates the nginx configuration file to match
- In Docker Compose (and local) environments, will deploy frontend-2 as default. As the frontend no longer provides nginx ingress, we need to deploy a separate nginx ingress.
- CircleCI builds and publishes the new nginx ingress

## To-do before merge:

- [x] Test changes locally
  - `docker compose -f docker-compose-deps.yml up -d`
  - `docker compose -f docker-compose-speckle.yml up -d`
  - visit `localhost:80` and FE2 should be displayed.
  - visit `localhost:80/graphql` and apollo should be displayed.
- [x] Update documentation to match new Docker Compose file

## Screenshots:



## Validation of changes:



## Checklist:



- [ ] My pull request follows the guidelines in the [Contributing guide](https://github.com/specklesystems/speckle-server/blob/main/CONTRIBUTING.md)?
- [ ] My pull request does not duplicate any other open [Pull Requests](../../pulls) for the same update/change?
- [ ] My commits are related to the pull request and do not amend unrelated code or documentation.
- [ ] My code follows a similar style to existing code.
- [ ] I have added appropriate tests.
- [ ] I have updated or added relevant documentation.

## References
